### DISCUSSION TITLE
The cli setup is not showing question for sync vs async anymore

### DISCUSSION THREAD

**Discussion by umair2020** [webdriverio/webdriverio#6775] [Q&A] [answered]
**Summary:**
On v7, the cli setup is not showing the question "**Do you want to run WebdriverIO commands synchronous or asynchronous?**". I think this is very important to be fix

**Comment by christian-bromann** [comment, marked as answer]
@umair2020 please see #6702, as long a we don't have a path forward yet, we can't allow people to start projects with sync when it might not be supported with Node.js v16 anymore.

**Reply by umair2020** [reply]
Thank you for a prompt reply! I will see what I can do when we move to v16 node. Currently, we are using v14

**Reply by christian-bromann** [reply]
Please follow the RFC for updates, the team will definitely try to keep this feature around.

**Reply by umair2020** [reply]
I did, thank you!

**Reply by umair2020** [reply]
@christian-bromann I want to know that if I can still run tests with sync mode on wdio v7. I actually installed v7 with wdio/cli and separately installed sync package
"@wdio/sync": "^7.5.2" 
and in wdio.conf I added sync: true
But seems like it is still expecting async/await in the code.
Can you tell if it is possible or not
